Connecting_MonteCarlo_to_ModernAI/README.md

```diff
@@ -22,3 +22,4 @@ Suggested time is 2-3 hours per notebook; total time for the module can be betwe
 |Notebook Title|Description|Data Science Learning Goals|Physics Learning Goal|
 |--------------|-----------|---------------------------|---------------------|
 |N1. Introduction to Monte Carlo Methods and Markov Chains | Introduces students to the difference between simulation and modeling, walks them through the steps of simulating a physical model and evaluating it.  Recommended to be assigned as homework before the next notebook.<br><br>Total Time: 1.5 hrs| DG1. Define the difference between a model and a simulation<br><br>DG2. Define a Markov Chain<br><br>DG3. Name three error metrics.|PG1. Given a physics equation, approximate the definite integral value through Monte Carlo integration. <br><br>PG2. Given a Markov Chain, simulate the state transitions using Monte Carlo sampling.<br><br>PG3. Calculate the error between a simulation and the analytical result.|
+|N2. 2D Ising Model | Introduces students to the derivation for the Ising Model, and walks them through a 2D simulation of a square Ising lattice.  Students are taught to identify the parts of the algorithm which are compute-intensive, and basic complexity and optimization skills for python code. <br><br> Total Time: | DG1. Implement the Metropolis-Hastings algorithm in Python using `numpy`.<br><br>DG2. Implement basic timing methods for understanding code execution. <br><br>DG3. Implement `numba` to improve code timing. | PG1. Understand the origin of the Ising Model and ferromagnetism. <br><br>PG2. Simulate net-magnetization curves as a function of temperature for a 2D system.<br><br> PG3. Connect the phase transition from magnetic disorder to order with the formation of magnetic domans in a thermalized system.|
```

Comment:
Could entries be added for the 3rd and 4th notebooks as well? Or if you are ultimately not including the 4th notebook, I think a couple updates are needed in the readme to clarify.

Reply:
Captured.

Reply:
Thank you for your comment!  I will update the README as notebooks become available.  The README has been updated to reflect the currently completed work.